src/main/java/com/stripe/model/LegalEntity.java

```diff
@@ -0,0 +1,164 @@
+package com.stripe.model;
+
+import com.stripe.exception.APIConnectionException;
+import com.stripe.exception.APIException;
+import com.stripe.exception.AuthenticationException;
+import com.stripe.exception.CardException;
+import com.stripe.exception.InvalidRequestException;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.List;
+import java.util.Map;
+
+public class LegalEntity extends APIResource {
+	String type;
+	Address address;
+	String businessName;
+	String businessTaxId;
+	String businessVatId;
+	DateOfBirth dob;
+	String firstName;
+	String lastName;
+	Address personalAddress;
+	String personalIdNumber;
+	String ssnLast4;
+	Verification verification;
+	List<Owner> additionalOwners;
+
+	public String getType() {
+		return type;
+	}
+	public Address getAddress() {
+		return address;
+	}
+	public String getBusinessName() {
+		return businessName;
+	}
+	public String getBusinessTaxId() {
+		return businessTaxId;
+	}
+	public String getBusinessVatId() {
+		return businessVatId;
+	}
+	public DateOfBirth getDob() {
+		return dob;
+	}
+	public String getFirstName() {
+		return firstName;
+	}
+	public String getLastName() {
+		return lastName;
+	}
+	public Address getPersonalAddress() {
+		return personalAddress;
+	}
+	public String getPersonalIdNumber() {
+		return personalIdNumber;
+	}
+	public String getSsnLast4() {
+		return ssnLast4;
+	}
```

Comment:
Removed `personal_id_number`, `ssn_last_4`, `business_tax_id`, and `business_vat_id`.